bedrock-devnet/devnet/__init__.py

```diff
@@ -126,18 +137,22 @@ def deploy_contracts(paths):
     account = response['result'][0]
     log.info(f'Deploying with {account}')
 
-    # send some ether to the create2 deployer account
-    run_command([
-        'cast', 'send', '--from', account,
-        '--rpc-url', 'http://127.0.0.1:8545',
-        '--unlocked', '--value', '1ether', '0x3fAB184622Dc19b6109349B94811493BF2a45362'
-    ], env={}, cwd=paths.contracts_bedrock_dir)
+    # Proxy exists on the fork public network(used by anvil)
+    # We don't need to deploy the proxy contract
+    # https://book.getfoundry.sh/tutorials/create2-tutorial
+    if not paths.fork_public_network:
```

Comment:
I think we have to need some ether to deploy some contracts although it is the fork public network. Is it resolved in the other way?

Reply:
Yes, anvil provides for us the default accounts for deploying some contracts, and we use `eth_accounts` function to get these accounts, work as the same way with `geth`

Reply:
Oh, I'm sorry. I understood it. Thank you for your kindness! 🙇🏼‍♂️